### ISSUE TITLE
Add ability to paste into the save dialog file name field

### ISSUE THREAD

**Issue by StephenOTT** [camunda/camunda-modeler#610]
Currently you cannot paste text into the save dialog's file name field.  (macos).

**Comment by philippfromme** [comment]
This is not an issue on Windows.

**Comment by nikku** [comment]
Is this still an issue with Camunda Modeler v1.16.2?

**Comment by StephenOTT** [comment]
Don't have latest due to older OS and the bug with the canvas. Unable to test.

**Comment by nikku** [comment]
I remember that issue. Thanks for the hint.

**Comment by nikku** [comment]
@pinussilvestrus Is this still an issue on `v3.0.0`? I remember we did stuff in that area.

**Comment by pinussilvestrus** [comment]
Indeed, this is still an issue (on macOS, is still not present on Windows). It might depend on the electron `showSaveDialog` implementation since we have [several other problems](https://github.com/camunda/camunda-modeler/issues/1221#issuecomment-465117635) with this topic.

**Comment by nikku** [comment]
@barmac @smbea Could you double check whether this is still an issue on MacOS?

**Comment by barmac** [comment]
This is still an issue in 5.8.0.
